### PR TITLE
added package constants

### PR DESCRIPTION
**Pull request by karollayna**
I changed functions common.always.available.stock.genomes and basic.offerings to package constants, as was requested in bioc review. 

@gladkia I have two questions: should I add getter functions for that constants or just leave as it is? Should I bump the version?

### CODE REVIEW

File: R/constants.R [hunk 1]
```diff
@@ -0,0 +1,12 @@
+COMMON.ALWAYS.AVAILABLE.STOCK.GENOMES <-
```
Comment (gladkia) [inline R/constants.R:1]: Please change to sth simpler without the dots, e.g. `cas_genomes` and `basic_genomes`.

**Comment by gladkia** [comment]
> I changed functions common.always.available.stock.genomes and basic.offerings to package constants, as was requested in bioc review.
> 
> @gladkia I have two questions: should I add getter functions for that constants or just leave as it is? Should I bump the version?
Please leave as it is and bump the version.

File: R/constants.R (hunk 1 above)
Comment (karollayna) [reply R/constants.R]: done

File: DESCRIPTION [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Package: igvShiny
 Title: igvShiny: a wrapper of Integrative Genomics Viewer (IGV)
-Version: 0.99.0
+Version: 0.99.1
```
Comment (gladkia) [inline DESCRIPTION:3]: Hmm, this version must stay (as required by the Bioc for the initial submission). @karollayna sorry for the confusion.

File: R/genomeSpec.R [hunk 3]
```diff
@@ -57,29 +37,20 @@ common.always.available.stock.genomes <- function() {
 #' @export
 #'
 currently.supported.stock.genomes <- function(test = FALSE) {
```
Comment (gladkia) [inline R/genomeSpec.R:39]: @karollayna, you are right. Changing the getter to sth with the verb in the name (e.g. `get_css_genomes`) makes sense.

File: DESCRIPTION (hunk 2 above)
Comment (karollayna) [reply DESCRIPTION:3]: no problem

File: R/genomeSpec.R (hunk 3 above)
Comment (karollayna) [reply R/genomeSpec.R:39]: ok

**Comment by gladkia** [comment]
@karollayna, please replace `currently.supported.stock.genomes` in:
- the vignette (`vignettes/igvShiny.Rmd`)
- demos (`inst/demos/stockGenomesDemo.R`)
- unit tests (comment in `inst/unitTests/test_genomeSpec.R`).

File: R/constants.R [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  c("hg38",
+    "hg19",
+    "mm10",
+    "tair10",
+    "rhos",
+    "custom",
+    "dm6",
+    "sacCer3")
+
+get_cas_genomes <- function() {
```
Comment (gladkia) [inline R/constants.R:14]: please move both these functions to R/genomeSpec.R and add documentation.

**Comment by gladkia** [comment]
Update test name in https://github.com/gladkia/igvShiny/pull/56/files#diff-038bac3018ee9c789a9fa109576e6b4ca143a20bc19e9e861ce66d897500f791L8.

File: R/constants.R (hunk 4 above)
Comment (karollayna) [reply R/constants.R]: all done

**Review by gladkia — APPROVED** [review]
GJ!

**Pull request merged**